src/components/iconButton/iconButton.stories.spec.tsx

```diff
@@ -14,9 +15,19 @@ const Wrapper = ({ children, inverse = false }: { children: ReactNode; inverse:
     {children}
   </div>
 );
+type Sizes = 'small' | 'medium' | 'large';
 const sizes: Sizes[] = ['small', 'medium', 'large'];
-const colors: Colors[] = ['neutral', 'white', 'mint', 'violet', 'ruby', 'gold', 'aqua', 'teal'];
-const tints: Tints[] = ['lightest', 'light', 'normal', 'dark', 'darkest'];
+const colors: (typeof COLORS[number] | 'white')[] = [
+  'neutral',
+  'white',
```

Comment:
There is no `white` color. We have a `neutral` color and 'lightest` tint.

Reply:
So remove white from all the places. Saw multiple places where we used it.